nix/docker.nix

```diff
@@ -8,18 +8,6 @@ let
   mkdir = name:
     runCommand "mkdir-${name}" { } "mkdir -p $out${lib.escapeShellArg name}";
 
-  mina-build-config = stdenv.mkDerivation {
-    pname = "mina-build-config";
-    version = "dev";
-    nativeBuildInputs = [ rsync ];
-
-    buildCommand = ''
-      mkdir -p $out/etc/coda/build_config
-      cp ${../src/config}/mainnet.mlh $out/etc/coda/build_config/BUILD.mlh
```

Comment:
Copying this source code into install dir make no sense to me in the first place.